install/share/profiles/KDCs_PKINIT_Certs.cfg

```diff
@@ -0,0 +1,109 @@
+profileId=KDCs_PKINIT_Certs
```

Comment:
Please comment on #62 for this part, this commit is pulled in only because #62 is a dependency and it is not commited to master yet.